src/components/Date/Date.tsx

```diff
@@ -0,0 +1,35 @@
+import "./Date.css";
+
+const MonthAndDayDate = ({
```

Comment:
Please rename this file! The rule is:

**The filename is the same as the component name!**

Also, another rule:

We never create components that are very similar in naming to built-in JavaScript classes/types. So, never ever have a component named `Array`, `Object`, `Date`, and so on...